gui.js

```diff
@@ -3703,6 +3714,7 @@ IDE_Morph.prototype.rawOpenProjectString = function (str) {
         );
     }
     this.stopFastTracking();
+    this.updateDocumentTitleText();
```

Comment:
I'm not sure, do we really need to set the document title again just because we're resizing a canvas element on the page?